crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -446,8 +459,17 @@ impl AuthorityPerEpochStore {
             cache_metrics,
             expensive_safety_check_config,
         );
+
+        let oauth_provider_jwk = tables
+            .load_oauth_provider_jwk()
+            .expect("Load oauth provider jwk at initialization cannot fail");
+
         let signature_verifier =
             SignatureVerifier::new(committee.clone(), signature_verifier_metrics);
+        for (_, v) in oauth_provider_jwk.iter() {
+            signature_verifier.insert_oauth_jwk(v);
```

Comment:
why is the key not inserted? i thought the signature verifier needs the entire map to verify?